spec/factories/custom_button_set.rb

```diff
@@ -2,5 +2,13 @@
   factory :custom_button_set do
     sequence(:name)        { |n| "custom_button_set_#{seq_padded_for_sorting(n)}" }
     sequence(:description) { |n| "custom_button_set_#{seq_padded_for_sorting(n)}" }
+
+    transient do
+      children []
+    end
+
+    after(:create) do |custom_button_set, evaluator|
+      custom_button_set.replace_children(evaluator.children)
+    end
```

Comment:
I thought it made [these](https://github.com/ManageIQ/manageiq/pull/14817/files#diff-9539fa63894852b868c79bfe4af0b82aR1441) a bit nicer (rather than having to do the `replace_children` after each one).